server/src/main/java/com/lunchtime/LunchTimeApplication.java

```diff
@@ -2,12 +2,28 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.security.servlet.SecurityAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.web.servlet.config.annotation.CorsRegistry;
+import org.springframework.web.servlet.config.annotation.WebMvcConfigurer;
 
-@SpringBootApplication
+@SpringBootApplication(
+    exclude = {SecurityAutoConfiguration.class, SecurityAutoConfiguration.class})
 public class LunchTimeApplication {
 
     public static void main(String[] args) {
         SpringApplication.run(LunchTimeApplication.class, args);
     }
 
+    /*Set the CORS configuration, more about CORS:
+      https://www.tutorialspoint.com/spring_boot/spring_boot_cors_support.htm */
```

Comment:
You don't need this comment. Your code speaks for himself.
https://gist.github.com/wojteklu/73c6914cc446146b8b533c0988cf8d29#comments-rules